lib/associations/mixin.js

```diff
@@ -148,54 +149,15 @@ Mixin.hasOne = singleLinked(HasOne);
 Mixin.belongsTo = singleLinked(BelongsTo);
 
 /**
- * Create an association that is either 1:m or n:m.
+ * Creates a 1:m association between this (the source) and the provided target. The foreign key is added on the target.
  *
- * ```js
- * // Create a 1:m association between user and project
- * User.hasMany(Project)
- * ```
- * ```js
- * // Create a n:m association between user and project
- * User.hasMany(Project)
- * Project.hasMany(User)
- * ```
- * By default, the name of the join table will be source+target, so in this case projectsusers. This can be overridden by providing either a string or a Model as `through` in the options.
-
- * If you use a through model with custom attributes, these attributes can be set when adding / setting new associations in two ways. Consider users and projects from before with a join table that stores whether the project has been started yet:
- * ```js
- * var UserProjects = sequelize.define('userprojects', {
- *   started: Sequelize.BOOLEAN
- * })
- * User.hasMany(Project, { through: UserProjects })
- * Project.hasMany(User, { through: UserProjects })
- * ```
- * ```js
- * jan.addProject(homework, { started: false }) // The homework project is not started yet
```

Comment:
Why remove this? It might not be the best fit at this exact point, but I still think its important to have some amount of examples in the api docs